components/cache.rst

```diff
@@ -177,8 +187,8 @@ Now you can create, retrieve, update and delete items using this cache pool::
 
 For a list of all of the supported adapters, see :doc:`/components/cache/cache_pools`.
 
-Advanced Usage (PSR-6)
-----------------------
+Advanced Usage
```

Comment:
@wouterj Should add `.. _cache-component-advanced-usage-psr-6:` here?

Reply:
Whenever I'm in doubt, I browse the current published article on symfony.com and copy the "anchor text". That's the reference we must use:

![rst-heading](https://user-images.githubusercontent.com/73419/55887680-38fc3280-5bae-11e9-8997-3f15df6e93b1.png)

Reply:
Lovely. Thank you!